Tasks/RunDistributedTests/RunDistributedTests.ps1

```diff
@@ -24,6 +24,40 @@ Function CmdletHasMember($memberName) {
     return $cmdletParameter
 }
 
+Function ModifyTestSettingsForDeploymentItems() {
+
+    if([System.String]::IsNullOrWhiteSpace($runSettingsFile) -Or ([string]::Compare([io.path]::GetExtension($runSettingsFile), ".runsettings", $True) -eq 0) -Or (Test-Path $runSettingsFile -pathtype container))
+    {
+        Write-Verbose "No modifications to test settings required"
+    }
+    else{
+        if (([string]::Compare([io.path]::GetExtension($runSettingsFile), ".testsettings", $True) -eq 0)) {
```

Comment:
is this applicable only for testsettings, if so can we move this condition to above